docs/modules/objectives/monument-modes.mdx

```diff
@@ -112,28 +125,60 @@ The `<modes>` tag can contain any number of `<mode>` tags with different duratio
     </tbody>
   </table>
 </div>
-Example
+
+#### Examples
+
+This specifies a mode that will change the material of affected objectives
+to gold block after 10 minutes of gameplay.
 
 ```xml
 <modes>
     <mode after="10m" material="gold block"/>
 </modes>
 ```
 
-This specifies a mode that will change the material of affected objectives to gold block after 10 minutes of gameplay.
+In order for an objective to change modes,
+it must have `modes` or `mode-changes="true"` in its element tag.
+Objectives with `mode-changes` enabled will cycle through all
+modes, regardless if the mode has an ID.
+
+```xml
+<destroyables name="Monument" materials="obsidian" mode-changes="true">
+    <!-- destroyables... -->
+    <!-- destroyables are already affected by modes listed in core -->
+</destroyable>
+<cores>
+    <core team="red" modes="gold-mode sandstone-mode glass-mode">
+        <region><cuboid min="10,15,12" max="12,13,16"/></region>
+    </core>
+</cores>
+```
+
+This specifies a mode that will change the material of affected objectives to
+coal blocks after 10 minutes of gameplay, with the following boss bar message
+in yellow, `> > > EASY MODE < < <`
 
 ```xml
 <modes>
     <mode after="5m" material="coal block" name="`eEASY MODE"/>
 </modes>
 ```
 
-This specifies a mode that will change the material of affected objectives to coal block after 10 minutes of gameplay, with the following chat message: `> > > EASY MODE < < <`
-
-`NOTE:` In order for an objective to change modes, it must have `mode-changes="true"` in its element tag.
+In this example, the two cores and the two monuments turn to glass in ten minutes, and the
+two cores will turn to gold in fifteen minutes. Enabling an objective's `mode-changes`
+attribute is not necessary in this senario.
```

Comment:
This bit is inconsistent with the previous example. Why does it say "ten" instead of "10"?